pkg/util/validation/limits.go

```diff
@@ -797,6 +814,22 @@ func (o *Overrides) SeparateMetricsGroupLabel(userID string) string {
 	return o.getOverridesForUser(userID).SeparateMetricsGroupLabel
 }
 
+func (o *Overrides) CostAttributionLabels(userID string) []string {
+	return o.getOverridesForUser(userID).CostAttributionLabels
+}
+
+func (o *Overrides) MaxCostAttributionLabelsPerUser(userID string) int {
+	return o.getOverridesForUser(userID).MaxCostAttributionLabelsPerUser
+}
```

Comment:
Is this a user setting? I can we make a fixed upper bound to this?

Reply:
capped to 4 in this commit and test updated 68410c8128c495897c224a11d2c8d2e22ee42069